src/app/features/search/multiple-search-results/multiple-search-results.component.html

```diff
@@ -4,6 +4,10 @@
   </div>
 
   <ng-container *ngFor="let vehicleTechRecord of vehicleTechRecords$ | async">
-    <app-single-search-result class="govuk-grid-column-full" [vehicleTechRecord]="vehicleTechRecord"></app-single-search-result>
+    <app-single-search-result
+      class="govuk-grid-column-full"
+      [vehicleTechRecord]="vehicleTechRecord"
+      *appUserRole="['CVSFullAccess', 'TechRecord.View']"
```

Comment:
Wouldn't this view be protected by the guard rather than hiding content which might be a bad user experience as they would think the view is broken?

Reply:
The route is guarded too, I took a both approch. It maybe worth considering if this is needed, especially as the API is never hit as it is protected too, but why would it ever be a bad UX? If you have the permission you will see it, if you don't then it doesn't matter. You need to be able to view a tech record to see it, this in my view is fine to hide it behind this.